File: php/kasirmini.php
<?php
$nama = readline("Masukkan nama barang: ");
$harga = floatval(readline("Masukkan harga barang: "));
$jumlah = intval(readline("Masukkan jumlah beli: "));

$total = $harga * $jumlah;

if ($total > 91000) {
    $diskon = $total * 0.01;
} else {
    $diskon = 0;
}

$total_bayar = $total - $diskon;

echo "\n===== STRUK DATA =====\n";
echo "nama barang    : $nama\n";
echo "harga satuan   : $harga\n";
echo "jumlah satuan  : $jumlah\n";
echo "total harga    : $total\n";
echo "diskon         : $diskon\n";
echo "total bayar    : $total_bayar\n";
echo "=========================\n";
?>

File: C/CPP/kasirmini.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string nama;
    double harga;
    int jumlah;

    cout << "Masukkan nama barang: ";
    getline(cin, nama);

    cout << "Masukkan harga barang: ";
    cin >> harga;

    cout << "Masukkan jumlah beli: ";
    cin >> jumlah;

    double total = harga * jumlah;
    double diskon;

    if (total > 91000) {
        diskon = total * 0.01;
    } else {
        diskon = 0;
    }

    double total_bayar = total - diskon;

    cout << "\n===== STRUK DATA =====\n";
    cout << "nama barang    : " << nama << endl;
    cout << "harga satuan   : " << harga << endl;
    cout << "jumlah satuan  : " << jumlah << endl;
    cout << "total harga    : " << total << endl;
    cout << "diskon         : " << diskon << endl;
    cout << "total bayar    : " << total_bayar << endl;
    cout << "=========================\n";

    return 0;
}
